kroxylicious-systemtests/src/test/resources/kafka-admin-client-template.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
What makes a Kubernetes a good fit for this use case?  Deployments are usually long-running and designed to be scaled.  I think Kubernetes Job would be a match better tool for this task.

Reply:
Done. Using jobs programmatically